modules/optimeraRtdProvider.js

```diff
@@ -132,6 +134,9 @@ export function init(moduleConfig) {
 
 /**
  * Set the score file url.
+ * This fully-formed URL for the data endpoint request to fetch
+ * the targeting values. This is not a js library, rather JSON
```

Comment:
Why does it have file extension .js

Reply:
This is a legacy extension on the backend that predates Prebid integration, which changed the contents of the file so Prebid could ingest the data. This is the same extension for the endpoint in our bidder adapter as well. The update is in our roadmap, but it is still a little ways away.

Reply:
Thanks for the explanation, I can see in the test response expected return looks like